argocd/overlays/moc-infra/applications/envs/moc/smaug/operate-first/opf-vault.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: opf-vault
+spec:
+  project: operate-first
+  source:
+    repoURL: https://github.com/operate-first/apps.git
+    targetRevision: HEAD
+    path: vault/overlays/moc/vault
```

Comment:
no such path exists, should be `vault/overlays/moc/smaug` (you'll need to create this)

Reply:
@HumairAK Good catch, updated path